client/lib/plans/plans-list.js

```diff
@@ -788,6 +790,8 @@ export const PLANS_LIST = {
 			] ),
 		getSignupFeatures: () =>
```

Comment:
This should have `FEATURE_ALL_PERSONAL_FEATURES_JETPACK` replaced with `FEATURE_ALL_FREE_FEATURES_JETPACK` when the personal-plan isn't enabled.